src/DataSource/DibiFluentDataSource.php

```diff
@@ -46,7 +46,7 @@ public function __construct(Fluent $dataSource, string $primaryKey)
 
 
 	/********************************************************************************
-	 *                          IDataSource implementation                          *
+	 *                          IDataSource implementation *
```

Comment:
Please revert, it is not symmetric this way :D

Reply:
Solved